index.html

```diff
@@ -59,5 +59,139 @@ <h3>I'am a Software Developer</h3>
         </ul>
       </div>
     </section>
+    <section id="works_section">
+      <div id="works_title">
+        <h1>My Recent Works</h1>
+      </div>
+      <div id="card_works">
+        <div class="work_model_1">
+          <div class="model_1-title">
+            <h1>Multi-Post Stories</h1>
+          </div>
+          <div class="model_1-description">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+          </div>
+          <div class="model_1-tech">
+            <ul>
+              <li><button>css</button></li>
+              <li><button>html</button></li>
+              <li><button>bootstrap</button></li>
+              <li><button>Ruby</button></li>
+            </ul>
+          </div>
+          <div class="model_1-project">
+            <button class="orange_button">See Project</button>
+          </div>
+        </div>
+        <div class="work_model_2">
+          <div class="model_2-title">
+            <h1>Profesional Art Printing Data</h1>
+          </div>
+          <div class="model-2-description">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <div class="model_2-tech">
+            <ul>
+              <li><button>html</button></li>
+              <li><button>bootstrap</button></li>
+              <li><button>Ruby</button></li>
+            </ul>
+          </div>
+          <div class="model_2-project">
+            <button class="orange_button">See Project</button>
+          </div>
+        </div>
+        <div class="work_model_2">
+          <div class="model_2-title">
+            <h1>Profesional Art Printing Data</h1>
+          </div>
+          <div class="model-2-description">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <div class="model_2-tech">
+            <ul>
+              <li><button>html</button></li>
+              <li><button>bootstrap</button></li>
+              <li><button>Ruby</button></li>
+            </ul>
+          </div>
+          <div class="model_2-project">
+            <button class="orange_button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="about_me">
+      <div id="about_me-title">
+        <h1>About me</h1>
```

Comment:
- [ ] In your `about me` section, Kindly adjust the distance between the header and the paragraph to match the original design.

![image](https://user-images.githubusercontent.com/98453979/212089193-3ea3b181-566b-49db-be43-3621990f9425.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/212089411-0102b562-3f9f-41f5-a540-260c5ec1f0fa.png)